docs/build_parameters.rst

```diff
@@ -0,0 +1,833 @@
+            }
+        },
+        "additionalProperties": false,
+        "required": ["api_url", "auth"]
+    },
+    "smtp": {
+        "description": "SMTP notifications",
+        "type": "object",
+        "properties": {
+            "host": {
+                "description": "SMTP host server name",
+                "type": "string"
+            },
+            "from_address": {
+                "description": "From email address in notification email",
+                "type": "string"
+            },
+            "additional_addresses": {
+                "description": "Email addresses to always send notification email",
+                "type": "array",
+                "items": {
+                    "type": "string"
+                }
+            },
+            "error_addresses": {
+                "description": "Email addresses to send notifications if no other email address can be resolved",
+                "type": "array",
+                "items": {
+                    "type": "string"
+                }
+            },
+            "domain": {
+                "description": "Domain used when constructing email addresses",
+                "type": "string"
+            },
+            "send_to_submitter": {
+                "description": "Send email notification to Koji task submitter",
+                "type": "boolean"
+            },
+            "send_to_pkg_owner": {
+                "description": "Send email notification to Koji package owner",
+                "type": "boolean"
+            },
+            "additionalProperties": false,
+            "required": ["url", "from_address"]
+
+
+        }
+    },
+    "pdc": {
+        "description": "Product Definition Center (PDC) instance",
+        "type": "object",
+        "properties": {
+            "api_url": {
+                "description": "PDC api url, including api version",
+                "type": "string"
+            },
+            "insecure": {
+                "description": "Don't check SSL certificate for api_url",
+                "type": "boolean"
+            }
+        },
+        "additionalProperties": false,
+        "required": ["api_url"]
+    },
+    "arrangement_version": {
+        "description": "Arrangement version",
+        "type": "integer"
+    },
+    "artifacts_allowed_domains": {
+        "description": "Domains allowed to use when fetching artifacts by url",
+        "type": "array",
+        "items": {
+            "type": "string"
+        }
+    },
+    "image_labels": {
+        "description": "Labels to be applied to container image",
+        "type": "object",
+        "properties": {},
+        "additionalProperties": true
+    },
+    "image_equal_labels": {
+        "description": "Labels that are expected to be equal in value",
+        "type": "array",
+        "items" : {
+            "type": "array",
+            "items": {
+                "type": "string"
+            }
+        }
+    },
+    "openshift": {
```

Comment:
TODO: When passed to worker builds, this must be overwritten somehow.

Reply:
Added the following to the section before schema definition where `orchestrate_build` plugin is mentioned:

> The plugin will also substitute any environment configuration if needed. As an example, the **openshift** section will be replaced with the worker OpenShift cluster information.